File: fizzbuzz.go
package main

import "fmt"

func main() {
  for n := 1; n <= 100; n++ {
    output := ""
    if n % 3 == 0 {
      output = output + "Fizz"
    }
    if n % 5 == 0 {
      output = output + "Buzz"
    }
    if output != "" {
      fmt.Println(output)
    } else {
      fmt.Println(n)
    }
  }
}

File: fizzbuzz.rb
#!/usr/bin/env -S ruby -w

(1..100).each do |n|
  output = "";
  if n%3 == 0
    output += 'Fizz';
  end
  if n%5 == 0
    output += 'Buzz';
  end

  print output != "" ? output : n, "\n";
end
